src/main/java/com/karankumar/bookproject/ui/FormErrors.java

```diff
@@ -0,0 +1,12 @@
+package com.karankumar.bookproject.ui;
+
+public class FormErrors {
```

Comment:
Thanks for refactoring this out into a new class. It's quite nice this way.

Since these error messages will only be used in the `BookForm`, could you move it into the ui/book package?

Reply:
Have changed the package but I was thinking of having a common errors file for all the forms.

Reply:
In my view, that would make sense only if there were common constants that were needed across the form classes.

Reply:
Sure, I had already changed the package.